bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/Clip2Thing.xml

```diff
@@ -11,6 +11,7 @@
 
 		<label>Hue Device</label>
 		<description>A Hue API v2 device with channels depending on its actual capabilities.</description>
+		<semantic-equipment-tag>LightSource</semantic-equipment-tag>
```

Comment:
Are you sure that `updateEquipmentTag` will override for **all** cases except `LightSource`? Perhaps it would be safer to remove it here, and then make sure to also set `LightSource` in that method, so that we would not risk wrongly setting `LightSource` for something which is not?

Reply:
Yes. That makes sense. I will do it. Although having a default in the XML is probably still a useful backup..

Reply:
It seems my point didn't get through, since there are now defaults at two levels: `LightSource` here and `Lightbulb` in code. So I believe the former will always be overwritten by the latter?

My point was the opposite: We should not have a default, except if there is something which can safely be assumed. Are you sure `Lightbulb` is such a safe "fallback"? I would rather have no default equipment tag than a security camera or something being tagged as a lightbulb. So no tag might be the sensible/safe default?

Reply:
Ok. I will remove the defaults.